src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -546,6 +551,29 @@ private static CommonCompilerOptions GetCompilationOptions(JObject rawObject, Pr
                 project.AnalyzerOptions = analyzerOptions;
             }
 
+            IncludeFilesResolver embed = null;
+            var embedOption = rawOptions.Value<JToken>("embed");
```

Comment:
save the `embedOption` to the `CommonCompilerOptions` instead of a resolver. delay create the resolver to later.